kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -128,6 +129,42 @@ def uninstall(
                 f"Release with name {release_name} not found. Could not uninstall app."
             )
 
+    def template(
+        self,
+        release_name: str,
+        chart: str,
+        values: dict,
+        flags: HelmTemplateFlags = HelmTemplateFlags(),
+    ) -> str:
+        """
+        Render chart templates locally and display the output.
+
+        From HELM: Render chart templates locally and display the output.
```

Comment:
perhaps remove this line since it's the same as above

Reply:
agreed, done